sig/handwritten/app/decorators/profile_decorator.rbs

```diff
@@ -0,0 +1,2 @@
+module ProfileDecorator : Profile
```

Comment:
The component after `:` is called `module-self-types`. It describes what classes or modules will include this module. It tells Steep to this module can use the methods of "includer" 's.
https://github.com/ruby/rbs/blob/master/docs/syntax.md#declarations

In this case, decorator modules can access the methods of the decorated object.